test/check-exports/test.mjs

```diff
@@ -0,0 +1,312 @@
+import { spawnSync } from 'child_process'
+import { deepStrictEqual } from 'assert'
+import { dirname, resolve } from 'path'
+import { fileURLToPath } from 'url'
+
+const cwd = dirname(fileURLToPath(import.meta.url))
+const hook = resolve(cwd, '..', '..', 'hook.mjs')
+
+const modules = [
```

Comment:
let's add a comment that this is the 250+ most popular packages on npm (and maybe even extract the web frameworks out to be a separate array for better readability).